.env.example

```
@@ -52,3 +52,21 @@ QUOTA_INBOX_LIMIT_COUNT=10000
+SLACK_APP_TOKEN=xapp-your-app-token
+SLACK_BOT_TOKEN=xoxb-your-bot-token
+SLACK_DEFAULT_CHANNEL=general
+SLACK_ENABLED=false
+SLACK_INCLUDE_ATTACHMENTS=true
+SLACK_NOTIFY_MENTION_FORMAT=agent_name
+SLACK_NOTIFY_ON_ACK=false
+SLACK_NOTIFY_ON_MESSAGE=true
+SLACK_SIGNING_SECRET=your-signing-secret
+SLACK_SYNC_CHANNELS=
+SLACK_SYNC_ENABLED=false
+SLACK_SYNC_PROJECT_NAME=Slack Sync
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)